ocs_ci/ocs/resources/pod.py

```diff
@@ -116,6 +116,13 @@ def get_fio_results(self):
                 ):
                     if sample:
                         return yaml.safe_load(self.fio_thread.result())
+            else:
+                # if self.fio_thread status is pending
+                for sample in TimeoutSampler(
```

Comment:
As the action done under 'else' here is the same as under the condition in line #111, you can add this to that condition:

if self.fio_thread and self.fio_thread.done() self.fio_thread.pending:

Reply:
self.fio_thread does not have pending attr

Reply:
Oh ok, it was a guess actually :)
Is there a way to do this condition then?

Reply:
self.fio_thread has only done and running, did not find any conditions. will dig more